pkg/reconciliation/constructor.go

```diff
@@ -31,7 +31,7 @@ func newPodDisruptionBudgetForDatacenter(dc *api.CassandraDatacenter) *policyv1.
 
 	pdb := &policyv1.PodDisruptionBudget{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        dc.LabelResourceName() + "-pdb",
+			Name:        dc.Name + "-pdb",
```

Comment:
Why use a `-pdb` suffix here ?
It seems useless to me, since K8S resource name only needs to be unique **for a given kind** of resource.